src/pidf_controllers/why_run_to_position_mode_is_not_a_good_idea.md

```diff
@@ -0,0 +1,57 @@
+# Why Run To Position Is Not A Good Idea
```

Comment:
This title could be better phrased.

"Not A Good Idea" is fairly weakly worded, I think a simple "Why Not Use RTP" or "Do Not Use RTP" is more direct and better.